### DISCUSSION TITLE
Flex alignSelf?

### DISCUSSION THREAD

**Discussion by ghost** [chakra-ui/chakra-ui#2798] [General]
Does the Flex component support alignSelf?

I don't see it in the documentation, but I tested it and it seems to work in Firefox but not Chrome.

Here's my code:

```
<Flex
	direction="column"
	h="100%"
	w="100%"
	justify="start"
	color="brand.blueDk"
	>
	<Checkbox size="lg" colorScheme="blue">
	  Remember Me
	</Checkbox>
	<Spacer />
	<Link fontSize="lg" alignSelf="end">
	  Forgot Password
	</Link>
</Flex>
```

And here's a screenshot of two browsers side by side:

![Chakra Flex AlignSelf](https://user-images.githubusercontent.com/55907892/102287726-437cdc80-3ef8-11eb-8343-0f5ec400e40e.png)

**Comment by with-heart** [comment]
Yeap, Chakra just outputs css so layout issues like this are at the browser level. In this case, `end` is not a value that Chrome supports in its box alignment model yet: https://stackoverflow.com/questions/59916581/align-items-end-difference-between-chrome-and-firefox

**Reply by ghost** [reply]
Thanks! Any suggestions on what properties to use to get the Forget Password right aligned within either a Flex box or Grid cell?

**Reply by ghost** [reply]
This works...

```
        <Flex dir="row">
          <Spacer />
          <Link fontSize="lg">Forgot Password</Link>
        </Flex>
```
